neptune/DESCRIPTION

```diff
@@ -17,4 +17,4 @@ BugReports: https://github.com/neptune-ai/neptune-r/issues
 Encoding: UTF-8
```

Comment:
I'm not sure that we're currently supporting the `del run['a']` aka. deleting a namespace/node. Do we?

Reply:
Ok. I see that there is something like `run['f/b'] = NULL` and it works as `del`